pkg/apis/app/v1alpha1/kogitoservices_types.go

```diff
@@ -225,7 +225,9 @@ func (k *KogitoServiceSpec) AddResourceLimit(name, value string) {
 func (k *KogitoServiceSpec) GetDeploymentLabels() map[string]string { return k.DeploymentLabels }
 
 // SetDeploymentLabels ...
-func (k *KogitoServiceSpec) SetDeploymentLabels(labels map[string]string) { k.DeploymentLabels = labels }
+func (k *KogitoServiceSpec) SetDeploymentLabels(labels map[string]string) {
```

Comment:
Please do not change this inline function.

Reply:
It changed due to code formating when I run `make build`, reverted it now

Reply:
done